modules/ROOT/pages/cloud-deployment-guide/editor-plugin-version.adoc

```diff
@@ -56,88 +56,88 @@ This channel deploys nightly builds of {productname}. This channel includes the
 
 ===== Example: Using the `{productmajorversion}-dev` release channel
 
-[source, html]
+[source, html, subs="attributes+"]
 ----
 <script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/{productmajorversion}-dev/tinymce.min.js" referrerpolicy="origin"></script>
 ----
 
 == Specifying the version of premium plugins deployed from Tiny Cloud
 
-Each {productname} version is bundled with a set of premium plugins, but it is possible to specify different versions of each premium plugin to use with {productname}. Use the URL query parameters to specify the version of each premium plugin to load. This approach works with both the link:{baseurl}/cloud-deployment-guide/editor-and-features[{productname} editor and premium plugins deployment via {cloudname}] or just the link:{baseurl}/cloud-deployment-guide/features-only[premium plugins deployment from {cloudname}].
+Each {productname} version is bundled with a set of premium plugins, but it is possible to specify different versions of each premium plugin to use with {productname}. Use the URL query parameters to specify the version of each premium plugin to load. This approach works with both the xref:cloud-deployment-guide/editor-and-features[{productname} editor and premium plugins deployment via {cloudname}] or just the link:{baseurl}/cloud-deployment-guide/features-only.adoc[premium plugins deployment from {cloudname}].
```

Comment:
This hasn't been converted properly, as you have a broken xref and another that has a link to an adoc file.

```suggestion
Each {productname} version is bundled with a set of premium plugins, but it is possible to specify different versions of each premium plugin to use with {productname}. Use the URL query parameters to specify the version of each premium plugin to load. This approach works with both the xref:cloud-deployment-guide/editor-and-features.adoc[{productname} editor and premium plugins deployment via {cloudname}] or just the xref:cloud-deployment-guide/features-only.adoc[premium plugins deployment from {cloudname}].
```